core/src/com/unciv/ui/screens/cityscreen/CityScreen.kt

```diff
@@ -259,7 +259,8 @@ class CityScreen(
             addWltkIcon("OtherIcons/WLTK 1") { color = Color.FIREBRICK }.padRight(10f)
         }
 
-        if (city.isPuppet) {
+        val canAnnex = !city.civ.getMatchingUniques(UniqueType.MayNotAnnexCities).any()
```

Comment:
here we can use hasUnique

Reply:
👍
They've all been replaced locally, am yet to commit

Reply:
Reminder :)

Reply:
Oops turns out I didn't replace them locally